src/main/kotlin/service/store/StoreService.kt

```diff
@@ -13,16 +13,18 @@ import service.user.balance.BalanceIncreaseRateManager.Companion.IncreaseRateSca
 import service.user.balance.BalanceManager
 import utility.PokeApi
 import utility.RedisConnector
+import java.math.BigDecimal
+import java.math.RoundingMode.CEILING
 import kotlin.random.Random
 
 const val BoosterpackSize = 5
 const val LocationIdBaseIncrease = 3.0
 const val SecondsInFiveMinutes = 300
-const val BoosterpackAmountLimit = 25
+const val BoosterpackAmountLimit = 55
```

Comment:
Ah, here it is. Maybe use it everywhere.

Reply:
It actually does not correspond, I just use 55 oftentimes for pseudo-random numbers :D